### PR TITLE
0.8.4 - Set auto fill option to "off"

### PR DESCRIPTION
**Pull request by osmarpetry**

### CODE REVIEW

File: src/core/InputBase.tsx [hunk 1]
```diff
@@ -33,6 +33,7 @@ const InputAdornment: SFC<IProps> = ({
     ...otherProps
```
Comment (williamrozin) [inline src/core/InputBase.tsx]:
```js
    style,
    autoComplete = 'off',
    ...otherProps
```

File: src/core/InputBase.tsx [hunk 2]
```diff
@@ -33,6 +33,7 @@ const InputAdornment: SFC<IProps> = ({
...
 }) =>
     <MuiInputBase
+        autoComplete="off"
```
Comment (williamrozin) [inline src/core/InputBase.tsx]: `autoComplete={ autoComplete }`

Comment (williamrozin) [reply src/core/InputBase.tsx]: same to `TextField.tsx` file

Comment (osmarpetry) [reply src/core/InputBase.tsx]: `'autoComplete' is assigned a value but never used.eslint(@typescript-eslint/no-unused-vars)`

File: src/core/InputBase.tsx [hunk 3]
```diff
@@ -30,9 +30,11 @@ const InputAdornment: SFC<IProps> = ({
     margin,
     padding,
     style = {},
+    autoComplete='off',
```
Comment (williamrozin) [inline src/core/InputBase.tsx]: spaces before and after `=`

**Pull request merged**